app/src/main/java/org/fossasia/badgemagic/adapter/SaveAdapter.kt

```diff
@@ -10,10 +10,10 @@ import androidx.appcompat.widget.AppCompatImageView
 import androidx.core.content.ContextCompat
 import androidx.recyclerview.widget.RecyclerView
 import com.google.android.material.chip.Chip
+import data.BadgeConfig
+import data.ConfigInfo
+import helpers.JSONHelper
```

Comment:
Why not prepending org.fossasia.badgemagic and using same package structure in common module as well?

Reply:
Yep, it would be perfect